langkit/__init__.py

```diff
@@ -14,11 +14,18 @@ class LangKitConfig:
     pattern_file_path: str = field(
         default_factory=lambda: _resource_filename("pattern_groups.json")
     )
+    response_pattern_file_path: Optional[str] = field(
+        default_factory=lambda: _resource_filename("pattern_groups.json")
+    )
```

Comment:
rather than duplicate the config properties for each field they might apply to, why don't we support more than one instance of LangKitConfig, each keyed to a column? so the structure would then be:
column_name -> language -> LangKitConfig

Reply:
Several of the column names are config driven. Not sure that would work.